From 6d11c43e07fc43a6111d582624b1e97ff5136c7c Mon Sep 17 00:00:00 2001
From: "Artur R. Czechowski" <arturcz@hell.pl>
Date: Sun, 18 Mar 2012 01:28:02 +0100
Subject: [PATCH] Lack of plugin during configure stage is critical error

When configure found there is no possibility to build the audacious plugin we
want to stop a compilation with error instead quietly just not building the
plugin.
---
 imms/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imms/configure.ac b/imms/configure.ac
index 6e34984..e0587f7 100644
--- a/imms/configure.ac
+++ b/imms/configure.ac
@@ -383,7 +383,7 @@ if test "$with_audacious" != "no"; then
 fi
 
 if test "$with_audacious" = "no"; then
-    AC_MSG_WARN([Not building audacious plugin])
+    AC_MSG_ERROR([Not building audacious plugin])
 else 
     AC_APPEND(PLUGINS, libaudaciousimms.so)
 fi
